modules/help_editor/test/help_editorTest.php

```diff
@@ -40,12 +40,12 @@ function setUp()
         $this->DB->insert(
             "help",
             array(
-             'helpID'   => '999999',
-             'parentID' => '-1',
-             'hash'     => $md5String,
-             'topic'    => 'Test Topic',
-             'content'  => 'This is a test content.',
-             'created'  => '2013-04-05 00:00:00',
+             'helpID'         => '999999',
+             'parentID'       => '-1',
+             'hash'           => $md5String,
+             'topic'          => 'Test Topic',
+             'projectContent' => 'This is a test content.',
```

Comment:
I think you removed the wrong column.. it should be `content` that remains, not `projectContent`.

Reply:
^ i agree with @driusan, every new help content goes into the 'projectContent' column but it should be in 'content'. For every existing entry, it looks like 'projectContent' has value null.